### ISSUE TITLE
move(to: ) forcing timeline view to 12:00 am 

### ISSUE THREAD

**Issue by drouston** [richardtop/CalendarKit#211]
### New Issue Checklist

- [x] Use the latest version of CalendarKit
- [x] I've read the [Contribution Guidelines](https://github.com/richardtop/CalendarKit/blob/master/CONTRIBUTING.md)
- [x] I've searched for [existing GitHub issues](https://github.com/richardtop/CalendarKit/issues)

### Issue Description

When I call dayView.state?.move(to: ) the subsequent timeline view resets to 12:00 am. I'd like for the position of the scroll view not to change, just as in the provided example. I am calling from within a delegate function of another library for a calendar cell selection. The scrollTo(hour24: ) and scrollToFirstEventIfNeeded() functions will work but only after the reset to 12:00 am. 

I'm using just a DayView rather than a DayViewController. I've removed the extraneous code (I think...).

Thanks.

##### Code I'm using with CalendarKit

```Swift
    @IBOutlet weak var dayView: DayView!

    override func viewDidLoad() {
        super.viewDidLoad()
        
        self.dayView.isHeaderViewVisible = false
        
        self.dayView.dataSource = self
        self.dayView.delegate = self
        self.dayView.reloadData()
        
    }

    override func viewDidAppear(_ animated: Bool) {
       
       self.dayView.scrollToFirstEventIfNeeded() //This scroll func works great
        
    }

    func calendar(didSelectDate date: Date) {
        
        self.dayView.state?.move(to: date)
        
    }

```

#### Result I am trying to achieve

See behavior in the iOS example.

**Comment by drouston** [comment]
Quick clarification: the iOS example has this behavior when 

autoScrollToFirstEvent = false

Thanks.

**Comment by drouston** [comment]
Updates:

- I set 

'''Swift
dayView.isHeaderViewVisible = true
'''

and attempted to tap the dates in the header view. I observed the same behavior with the day view resetting to 0:00 hrs.

- I've removed and reinstalled the pods for CalendarKit and dependencies for a clean install. No change.

- I attempted to modify the date in move(to:) to include hour and minute components equal to the previous date. No change.

The swipe gesture on the day view maintains the same time as the previous page view. My guess at this point is this has something to do either with using the DayView as a subview or that I've setup the view controllers in Interface Builder.

**Comment by drouston** [comment]
<div style='position:relative; padding-bottom:calc(216.43% + 44px)'><iframe src='https://gfycat.com/ifr/VagueAcademicEland' frameborder='0' scrolling='no' width='100%' height='100%' style='position:absolute;top:0;left:0;' allowfullscreen></iframe></div>

**Comment by drouston** [comment]
Hello @richardtop, just following up on this one. Any news?

Thanks.

**Comment by richardtop** [comment]
This is a known bug in the library. Thank you for reporting it.
